CMake/hoomd/FindCUDALibs.cmake

```diff
@@ -49,6 +49,7 @@ if (HIP_PLATFORM STREQUAL "nvcc")
         INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"
       )
     endif()
+    find_package_message(CUDALibs "Found nvrtc: ${CUDA_nvrtc_LIBRARY}" "[CUDA_nvrtc_LIBRARY]")
```

Comment:
I assume that you are using CUDALibs rather than the specific library/program/package in these messages on puprose?

Reply:
On testing further I found that the first argument to `find_package_message` needs to be unique for each library to avoid repeated messages: https://cmake.org/cmake/help/latest/module/FindPackageMessage.html. See the fix in 64cbf00.